src/Slices/ServiceInstanceDetails/UI/Components/AttributesComponents/AttributesEditor.tsx

```diff
@@ -48,8 +77,63 @@ export const AttributesEditor: React.FC<Props> = ({
     setSelectedSet(value);
   };
 
+  /**
+   * useCallback hook onEditorUpdate
```

Comment:
We could make a note when we are using callbacks, it's the performance optimization?

Reply:
This is because the method is used in a UseEffect dependency, and would cause too many rerenders if not wrapped in a useCallback hook. This is a React practice, and gets flagged by the linting if ommited. I will add this comment in the code, but this is general knowledge about React.

Reply:
I don't know the case here, but they mention that if withouth use callback it won't work that means there is some underlying issue, but it couldn't be the case here so I asked